### PR TITLE
Paid Stats: Add Jetpack Stats to plans pages

### PR DESCRIPTION
**Pull request by DustyReagan**
Related to https://github.com/Automattic/dotcom-forge/issues/4595

> [!CAUTION]
> These updates are not behind a feature flag, so let's wait to merge this until paid-stats has gone live.

## Proposed Changes

* This PR adds "Jetpack Stats" as a feature to the Explorer plan in these plan comparison pages:
  * /plans/:site and /start/plans
  * Explorer checkout page
  * Link-in-bio flow
  * Blog onboarding flow
  * Newsletter flow
  * The comparison table at the bottom of /plans/:site and /start/plans

Copy:
* The proposed feature name is "Jetpack Stats"
* The proposed feature description (when your cursor hovers over the feature name) is "Finesse your scaling up strategy with detailed insights and data."

## Testing Instructions

* Load this PR in localhost or Calypso Live
* View the feature in each plan page above and hover over it to view the description copy. You can find screenshots to compare with in the code comments below.
* Ensure there are no new console errors.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/calypso-products/src/plans-list.tsx [hunk 1]
```diff
@@ -1491,6 +1493,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_PREMIUM_THEMES_V2,
 		FEATURE_WORDADS,
 		FEATURE_STYLE_CUSTOMIZATION,
+		FEATURE_STATS_PAID,
```
Comment (DustyReagan) [inline packages/calypso-products/src/plans-list.tsx:1496]: This is the checkout page for the Explorer plan.
Screenshot 2024-01-05 at 1 40 10 PM

File: packages/calypso-products/src/plans-list.tsx [hunk 2]
```diff
@@ -1483,6 +1484,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
...
```
Comment (DustyReagan) [inline packages/calypso-products/src/plans-list.tsx]: This is blog onboarding plans page.
Screenshot 2024-01-05 at 3 04 06 PM

File: packages/calypso-products/src/plans-list.tsx [hunk 3]
```diff
@@ -1445,6 +1445,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_STYLE_CUSTOMIZATION,
 		FEATURE_LIVE_CHAT_SUPPORT,
 		FEATURE_WORDADS,
+		FEATURE_STATS_PAID,
```
Comment (DustyReagan) [inline packages/calypso-products/src/plans-list.tsx]: This is /plans/:site page and also /start/plans
Screenshot 2024-01-05 at 3 08 37 PM

File: packages/calypso-products/src/plans-list.tsx [hunk 4]
```diff
@@ -1437,6 +1438,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_VIDEOPRESS_JP,
 		FEATURE_UNLTD_SOCIAL_MEDIA_JP,
 		FEATURE_WORDADS,
+		FEATURE_STATS_PAID,
```
Comment (DustyReagan) [inline packages/calypso-products/src/plans-list.tsx:1441]: This is the link-in-bio flow.
link-in-bio

File: packages/calypso-products/src/plans-list.tsx [hunk 5]
```diff
@@ -1413,6 +1413,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_PREMIUM_THEMES_V2,
 		FEATURE_UNLTD_SOCIAL_MEDIA_JP,
 		FEATURE_VIDEOPRESS_JP,
+		FEATURE_STATS_PAID,
```
Comment (DustyReagan) [inline packages/calypso-products/src/plans-list.tsx:1416]: This is the Newsletter flow.
newsletter

File: packages/calypso-products/src/feature-group-plan-map.ts [hunk 6]
```diff
@@ -225,6 +226,7 @@ export const featureGroups: Partial< FeatureGroupMap > = {
 			FEATURE_NEWSLETTERS_RSS,
 			FEATURE_AD_FREE_EXPERIENCE,
 			FEATURE_WORDADS,
+			FEATURE_STATS_PAID,
```
Comment (DustyReagan) [inline packages/calypso-products/src/feature-group-plan-map.ts:229]: This is the comparison chart at the bottom of /plans/:site page and also /start/plans
plans-comparison

**Comment by candy02058912** [comment]
Updated to accommodate for the feedback p1704703628817349/1704490274.145559-slack-C068X5XJT3J

`/plans/:site`
planssite

**Review by candy02058912 — APPROVED** [review]
Looks good to me! I moved the Jetpack Stats to the Jetpack section for the main plans grid (/plans/:site). So would be great to get another review on your end to double-check!

Also tested in `/setup/link-in-bio-tld/domains?new=test.link`

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/10748936

Some locales (Hebrew) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @DustyReagan, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Finesse your scaling up strategy with detailed insights and data.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/10748936).

**Pull request merged**